test/integration/targets/postgresql/tasks/main.yml

```diff
@@ -724,6 +724,28 @@
       - "result_database.stdout_lines[-1] == '(1 row)'"
       - "'{{ db_user2 }}=Cc' in result_database.stdout"
 
+- name: Grant execute to all function privileges
+  postgresql_privs:
+    type: "function"
+    state: "absent"
+    privs: "EXECUTE"
+    roles: "{{ db_user2 }}"
+    objs: "ALL_IN_SCHEMA"
+    db: "{{ db_name }}"
+    login: "{{ db_user1 }}"
+    password: "password"
+    host: "localhost"
+
+- name: Check that all functions have no execute privileges
+  become: True
+  become_user: "{{ pg_user }}"
+  shell: psql {{ db_name }} -c "SELECT proacl FROM pg_proc"
+  register: result_database
+- assert:
+    that:
+      - "'=X/{{ db_user2 }}' in item"
+  with_items: "{{ result_database.stdout_lines[:-1] }}" # remove last line
```

Comment:
CI failure in integration tests: https://app.shippable.com/github/ansible/ansible/runs/112029/50/tests

Also, this needs to be rewritten to avoid executing the task once for every line in `result_database`.

Reply:
How would you suggest I rewrite it? This is the way I would know how to do it

Reply:
Perhaps changing the shell command to something like:

```yaml
shell: "psql {{ db_name }} -c "SELECT proacl FROM pg_proc" | grep -v '=X/{{ db_user2 }}'"
```

Then have a single (non looped) assert check the results of that.